### ISSUE TITLE
Feature: Add setting to enable/disable the copy to clipboard during quick preview

### ISSUE THREAD

**Issue by TripSixes** [farkam135/GoIV#693]
As mentioned on a reddit post, this feature request is to add a setting that allows the user to enable or disable the automatic copying to clipboard after quick preview.  This would be disabled by default.

**Comment by nahojjjen** [comment]
So I tried implementing this (was in [this ](https://github.com/farkam135/GoIV/commit/b4395f7f65f79604bcf8ef266cdcd15665becdc0)bad commit), and here are my thoughts:

I didn't really have a usecase for it where the feature was really useful for me, I went over about 30 pokemon, and the initial quick iv preview popup made me toss about 25 of them, the remaining 5, I wanted to keep. This is where the clipboard on iv-preview would in theory be useful, however these 5 pokemon are also the pokemon interesting enough to go through appraisal refinement, which means that the copy to clipboard during quick preview wasnt helping.

On the other hand, it actually led to an issue. Once I had gone through appraisal narrowing and gotten a more exact clipboard result, I exited out of the goiv overlay to rename my pokemon, however clicking the screen triggered the quickIV preview again, which overwrote my more specific clipboard.

So to summarize, unless someone can come up with a good use-case scenario for this feature & how to solve the problem of appraise-narrowed results getting overwritten without messy code, I dont think this feature would be a good idea.

**Comment by TripSixes** [comment]
I agree with your assessment.  I also agree that this functionality is rarely needed unless you are narrowing down the results using appraisal info.. and that any code that would need to be added to "don't auto-copy if user just now appraised and has more precise info" would be difficult to get working flawlessly.

Will go ahead and close this issue.
